### PR TITLE
Add AMQP support

### PR DESCRIPTION
**Pull request by mertyildiran**

### CODE REVIEW

File: tests/configs/yaml/hbs/amqp/key.pem [hunk 1]
```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (haiut) [inline tests/configs/yaml/hbs/amqp/key.pem:1]: :exclamation:**Cycode**: Secret of type: 'Private Key' was found.

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |

Comment (undera) [reply tests/configs/yaml/hbs/amqp/key.pem:1]: #cycode_secret_ignore_here

Comment (haiut) [reply tests/configs/yaml/hbs/amqp/key.pem:1]: :+1: Cycode has finished processing your request. Please review your status checks.

File: docs/Async.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Asynchronous Actors for Message Bus
 
-Mockintosh offers "Mock Actor" approach for using with asynchronous message bus technologies, such as Kafka. "Mock
-Actor" approach requires you to provide deployed message bus instance, and configure valid address for it inside
+Mockintosh offers "Mock Actor" approach for using with asynchronous message bus technologies, such as
```
Comment (undera) [inline docs/Async.md:3]: I would explicitly mention RabbitMQ here, to help people understand that their async technology is supported.

**Pull request merged**